packages/did/src/DidResolver/DidResolver.ts

```diff
@@ -44,7 +50,9 @@ export async function resolve(
 
   // If the full DID has been deleted (or the light DID was upgraded and deleted),
   // return the info in the resolution metadata.
-  const isFullDidDeleted = await queryDidDeletionStatus(did)
+  const isFullDidDeleted = !(
+    await api.query.did.didBlacklist.hash(didToChain(did))
+  ).isEmpty
```

Comment:
honestly we may just want to go back to 

```suggestion
  const isFullDidDeleted = (
    await api.query.did.didBlacklist(didToChain(did))
  ).isNone
```

just bc it's easier to explain and to read and our code now has to serve as an example for everyone for how to use the api object

Reply:
Addressed in https://github.com/KILTprotocol/sdk-js/pull/634